### PR TITLE
Added logo to README.rst

### PR DESCRIPTION
**Pull request by lk-geimfari**
I added logos in two color schemes:

First is:
<p align="center">
      
</p>

Second is:
<p align="center">
      
</p>

Images are located in media/

In README.rst we use first logo, but i can change it, if you want.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-=====================
+.. raw:: html
+
+    <a href="https://github.com/sobolevn/django-split-settings">
+      <p align="center">
+         <img src="https://i.imgur.com/vISkpwj.png">
+      </p>
+    </a>
+
+----------
```
Comment (lk-geimfari) [inline README.rst]: Now, when i see how its looks i'm not sure that is line is cool. Should I delete it?

**Comment by sobolevn** [comment]
I like both, but the first one is slightly better.

File: README.rst (hunk 1 above)
Comment (sobolevn) [inline README.rst]: `rst` looks ugly anyways.

**Comment by lk-geimfari** [comment]
@sobolevn I removed second logo. Can i merge it or any other remarks?

File: README.rst [hunk 2]
```diff
@@ -1,6 +1,12 @@
-=====================
-django-split-settings
-=====================
+.. raw:: html
+
+    <a href="https://github.com/sobolevn/django-split-settings">
+      <p align="center">
+         <img src="https://i.imgur.com/vISkpwj.png">
```
Comment (sobolevn) [inline README.rst]: This link points to `imgur`. Should not it point to our logo?

Comment (lk-geimfari) [inline README.rst]: @sobolevn Oh, sorry. I forget to add link to uploaded logo. One minute, please.

**Comment by lk-geimfari** [comment]
@sobolevn [Here](https://github.com/sobolevn/django-split-settings/tree/logo) i renamed `media/` to `other/`. I can create PR and merge it if name of directory is significant or remove branch if not.

**Comment by sobolevn** [comment]
`media` is fine. Thanks!

**Pull request merged**